inventory/hosts.example

```diff
@@ -585,6 +585,11 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 # openshift_storageclass_name=gp2
 # openshift_storageclass_parameters={'type': 'gp2', 'encrypted': 'false'}
 #
+# PersistentLocalStorage
```

Comment:
example hosts should include `openshift_persistentlocalstorage_path` (with default value)